webpack.config.js

```diff
@@ -38,7 +39,10 @@ module.exports = env => {
     {
       target: "electron-main",
       entry: "./src/main/index.ts",
-      output: { filename: `${distDir}/main.js` },
+      output: {
+        path: distDir,
+        filename: `main.js`
```

Comment:
You should use double quote.

Reply:
https://github.com/bdash-app/bdash/pull/62/commits/2285efaff60601511b2f5f8a83536980dff20c66